### ISSUE TITLE
OpenTTD doesn't detect music when put in the baseset directory

### ISSUE THREAD

**Issue by rlelchev** [OpenTTD/OpenMSX#29]
I installed OpenSFX and I thought the installation for OpenMSX will be similar. I put everything where it must be, but when I open the settings and click on the drop down menu I can't find OpenMSX. It is just missing from the list. Any help?

**Comment by glx22** [comment]
So I guess you just put `openMSX.tar` in baseset directory. But for MSX to work, the tar needs to be extracted.

**Comment by rlelchev** [comment]
I didn't even know that! Thanks!

**Comment by sidesw1pe** [comment]
I realise this is closed, but I just installed OpenTTD and encountered this issue, and felt like including a comment.
In the readme.txt, for the manual installation, it specifically states:
> Unpack the zip file into the OpenTTD 'baseset' directory

The zip file contains the tar file. Therefore I extracted the tar file.
To install the sound and graphics addons you just need to extract the tar. 

_Maybe the documentation could be more specific?_
